extensions/ql-vscode/src/vscode-tests/minimal-workspace/databases/db-panel.test.ts

```diff
@@ -443,6 +443,91 @@ describe("db panel", () => {
     }
   });
 
+  describe("addNewRemoteRepo", () => {
+    it("should add a new remote repo", async () => {
+      const dbConfig: DbConfig = createDbConfig({
+        remoteRepos: ["owner1/repo1"],
+      });
+
+      await saveDbConfig(dbConfig);
+
+      const dbTreeItems = await dbTreeDataProvider.getChildren();
+
+      expect(dbTreeItems).toBeTruthy();
+      const items = dbTreeItems!;
+
+      const remoteRootNode = items[0];
+      const remoteRepos = remoteRootNode.children.filter(
+        (c) => c.dbItem?.kind === DbItemKind.RemoteRepo,
+      );
+      const repo1 = remoteRootNode.children.find(
+        (c) =>
+          c.dbItem?.kind === DbItemKind.RemoteRepo &&
+          c.dbItem?.repoFullName === "owner1/repo1",
+      );
+
+      expect(remoteRepos.length).toBe(1);
+      expect(remoteRepos[0]).toBe(repo1);
+
+      await dbManager.addNewRemoteRepo("owner2/repo2");
```

Comment:
These tests were copied from `describe("addNewList",`. I'm concerned we're calling the `dbManager` directly instead of using the `db-panel` (as it's a db-panel test).

Is it possible that we should move these tests into the dbManager-test-file?..

Reply:
You're right: this test file doesn't actually test the panel, rather the manager, config store, tree data provider, and the interactions between them 🤔 

It probably makes sense to restructure/rename this test file, but that might be a question for a future PR 😅 Happy to keep this as-is for now.